data/json/items/book/rifle.json

```diff
@@ -60,7 +60,7 @@
       },
       {
         "id": "manual_rifle_1",
-        "name": { "str": "SAS and Elite Forces Guide Sniper", "str_pl": "copies of SAS and Elite Forces Guide Sniper" },
+        "name": { "str": "SAS and Elite Sniper Forces Guide", "str_pl": "copies of SAS and Elite Sniper Forces Guide" },
```

Comment:
It is not a name of a book in the comment, isnt it? But it sound silly in this order, i agree

Reply:
Apologies about the book variant title, but I neglected to double check it with the actual source and changed it as it sounded somewhat silly. Would you rather I change it back to its original form?